packages/web/README.md

```diff
@@ -23,21 +23,12 @@ npm i --save @naturacosmeticos/natds-web
 yarn add @naturacosmeticos/natds-web
 ```
 
```

Comment:
We should explain how to use our lib on jsdelivr unpkg. We should be clear that jsdelivr is our recommended way since it have more pops and CDN/DNS redundancy. We should link to the explanation of jsdelivr network here: https://www.jsdelivr.com/network

Reply:
We should also add it to our readme on the root folder... maybe?